.github/workflows/tests-integration.yml

```diff
@@ -65,18 +65,9 @@ jobs:
             --ignore tests/integration_tests/allennlp_tests \
```

Comment:
AllenNLP seems to support for Python 3.6. Do we need to exclude the AllenNLP related tests here?
https://pypi.org/project/allennlp/

> Requires: Python >=3.6.1

```suggestion
```